Git Testing/Scenes/Details/View/PictureDetailVC.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PictureDetailViewController.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 14/01/2021.
+//
+
+import UIKit
+
+class PictureDetailVC: UIViewController {
+
+    @IBOutlet weak var theImageView: UIImageView!
+    
+    var picture: Picture! {
+        didSet{
+//            _ = PictureDetailViewModel(picture: picture, viewController: self)
+        }
+    }
+    
+    override func viewDidLoad() {
```

Comment:
Put a mark above this method too. 
Every method, property, actions and outlets should be under some MARK.
You can mark these methods like `MARK: ViewController Lifecycle Methods`